src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -143,7 +150,55 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     if (onTabSelected) {
       onTabSelected(index);
     }
-    setCurrentIndex(index);
+    setCurrentActiveIndex(index);
```

Comment:
I think clicked tab should also be focused. I do that in date picker and also thats how w3 works. Now when you click with mouse and use arrow on keyboard, active tab firstly gets focused (you do not move...).

Reply:
And another thing, the styling... When you select tab with keyboard, outline should not be visible (act same as mouse click).
![image](https://user-images.githubusercontent.com/81580355/126292800-210609b1-5679-433d-a6f4-877931a72dac.png)

Reply:
clicked tab also updates focused index now, good catch.

Styling looks fine to me, but if we want to update, it would go in css.